app/views/pregnancies/_call_log.html.erb

```diff
@@ -1,5 +1,5 @@
 <section id="call_log" class="call-info tab-pane">
-	<div class="margin-bottom col-md-12">
+	<div class="margin-bottom col-sm-12">
 		<h1 class="border-bottom title">Call Log</h1>
```

Comment:
There's a merge conflict here as well because we're using an h2 for 'Call Log' now